src/formattingService/formattingService.ts

```diff
@@ -33,9 +33,12 @@ import * as stringExtensions from "./../stringExtensions";
 import { findDateFormat, fixDateTimeFormat } from "./../formatting";
 import { IFormattingService, DateTimeUnit } from "./iFormattingService";
 import powerbi from "powerbi-visuals-tools";
-import * as Globalize from "globalize";
+import * as Globalize from "globalize/lib/globalize";
 import * as Cultures from "globalize/lib/cultures/globalize.cultures";
 
+// to prevent removing loading globalize.cultures file by optimizer
+console.log(Cultures);
```

Comment:
Can we import `Cultures` via ProvidePlugin?

Reply:
In this case, tools will import into each visual. For visuals without formatting utils too.